.bit/responses/4.3-Week 4 Step 3.md

```diff
@@ -1,32 +1,63 @@
+- [ ] Commit your code to `bunnimage/index.js`
 
-Once your MVP has been completed, it's time to finish the final project! 
+### 🚧 Test your Work
+Open up your LiveServer plugin and start your local server. **To test your web app:**
 
-In addition to the project, **write a blog tutorial** that describes step-by-step how a beginner could go around recreating your project.
+:one: Upload an image and click submit. *Do you get a "Your image has been stored successfully!" message?*
 
-The outline of your blog tutorial should look like the following:
+:two: Log into your Azure portal and navigate to your storage account. *In your container, do you see your image?*
 
-1. **Abstract:** A 200-250 word description of your project including why you made this project, what problem it solves, what your solution is, and what impact it has on the world.
-2. **Image** of yourself working on the project (candid or posed, as high quality as possible)
-3. **Profile Picture** (also as high quality as possible)
-4. **Tag line:** An IMPACTFUL description of your project (with a maximum of one sentence)
-5. **Step-by-step blog**
-    - **Beginning:** why you made this project, what problem it solves
-    - **Middle:** clear instructions on how to make this project from scratch, split code up into small parts and explain each part concisely but precisely
-    - **End:** what impact does this project have, what would you like to do to improve it, how you feel about this project, etc.
-    - **Tip:** use simple language and a lot of visuals, make it easy to read and follow
+## How do I call the Azure Function?
 
-To write your blog tutorial, it might be a good idea to use a markdown editor like [Typora](https://typora.io/).
+The Fetch API is built in to Javascript to make HTTP requests (using GET, POST and other methods), download, and upload files.
 
-**To move on, commit your final project WITH a markdown file of your blog tutorial. Comment any changes you made to your final project in comparison to your project proposal!**
+To start a request, call the special function `fetch()`:
+
+```javascript 
+const response = await fetch(resource, options);
+```
+
+`Fetch` accepts two arguments:
+- `resource`: the URL string, or a Request object
+- `options`: the configuration object with properties like method, headers, body, credentials, and more.
+
+<details>
+<summary>:question: What does fetch exactly do?</summary>
+  </br>
+
+`fetch()` starts a request and returns a promise. When the request completes, the promise is resolved with the Response object. **If the request fails due to some network problems, the promise is rejected.**
+> Keep this in mind if your function throws a promise error!
+  <br><br/>
+</details>
+
+## How to configure "options"
+In order to pass the type of method, the payload in the pody, and headers, you have to configure the "options" part of the fetch function. 
```

Comment:
Would be nice to add quick links to documentation and explain the difference between `node-fetch` and `fetch`